OpenRA.Game/Map/Map.cs

```diff
@@ -335,8 +335,8 @@ public Map(ModData modData, IReadOnlyPackage package)
 			if (MapFormat != SupportedMapFormat)
 				throw new InvalidDataException($"Map format {MapFormat} is not supported.\n File: {package.Name}");
 
-			PlayerDefinitions = MiniYaml.NodesOrEmpty(yaml, "Players");
-			ActorDefinitions = MiniYaml.NodesOrEmpty(yaml, "Actors");
+			yaml.TryGetNodes("Players", out PlayerDefinitions);
```

Comment:
These will need to be
```csharp
if (!yaml.TryGetNodes("Players", out PlayerDefinitions))
	PlayerDefinitions = new List<MiniYaml>();
```
to preserve the current behaviour, which is probably required to not crash.